.gitignore

```diff
@@ -4,3 +4,6 @@
 xcuserdata/
 
 Carthage/Build/
+
+.DS_Store
+Sources/.DS_Store
```

Comment:
A .DS_Store is enough. Also it would be better adding a newline at end of the file.

Reply:
Thanks for the catch. updated now.